e2e/workspace/server_example/BUILD.bazel

```diff
@@ -0,0 +1,22 @@
+load("@aspect_rules_cypress//cypress:defs.bzl", "cypress_test")
+load("@aspect_rules_js//js:defs.bzl", "js_binary")
+
+js_binary(
+    name = "server",
+    data = ["//:node_modules/express"],
+    entry_point = "server.js",
+)
+
+cypress_test(
+    name = "server_example",
+    args = [
+        "run",
+        "--config-file=cypress.config.js",
```

Comment:
I'm surprised this doesnt need some `$(execpath)` or `$(rootpath)` or `path.join(package_name()`

Reply:
Yeah, I made the default behavior of the cypress rules be to change the cwd to the directory where the target was defined. Its a convenience for sure, but maybe confusing in hindsight.